src/datascience-ui/ipywidgets/renderer/index.ts

```diff
@@ -2,31 +2,31 @@
 // Licensed under the MIT License.
 
 import './styles.css';
-import { NotebookOutputEventParams, NotebookRendererApi } from 'vscode-notebook-renderer';
+//import { NotebookOutputEventParams, NotebookRendererApi } from 'vscode-notebook-renderer';
```

Comment:
This is the old build of the renderer. I left it in, just commented out for now in case on Monday we have a better idea for doing this. If we think the hack is staying around in some form I'll pull it.